Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Application/Services/StringRandomizerService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MvvmSeed.Application.Data;
+using MvvmSeed.Domain.Services;
+using Realms;
+using System;
+using System.Linq;
+
+namespace MvvmSeed.Application.Services
+{
+    /// <summary>
+    /// Simple implementation of <see cref="IStringRandomizerService"/> that stores randomization info on a local database (Realm)
+    /// </summary>
+    public class StringRandomizerService : IStringRandomizerService
+    {
+        //TODO - When attempting to write UTs for this service, found out I couldn't because Realm class doesn't implement an interface/virtual methods. 
```

Comment:
I think this might be a showstopper for this seed part. If it does not allow for testing, it will become an issue to the projects generated from this seed.